frb_codegen/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 categories.workspace = true
 description.workspace = true
 edition.workspace = true
+rust-version = "1.74.0"
```

Comment:
It may be great if we can lower the MSRV by fixing some trivial compile errors ;)

Reply:
Hmmm - I personally don't mind, as I like to use the latest version.  
But I suggest opening another issue for that. In #1454 you can get an impression of what kind of changes would have to be made (that was with rust 1.71., the compiler complained about `pup (crate)` mostly). 
I assume that would require a redesign of the modularization.  
IMHO: Not worth it, would not do that unless somebody complains.

I got to the MSRV of 1.74.0 by using `cargo msrv`, which performs a bisect search. So it won't compile to anything lower without changes to the code.
I would leave it as it, and only of somebody raises an issue needing a lower version work on specifically supporting from that version on.

Reply:
> I suggest opening another issue for that

Sure! Then let's keep this PR simple and atomic.